sdcm/sct_config.py

```diff
@@ -169,1450 +186,1774 @@ def boolean(value):
-             type=str,
-             help="Branch of scylla manager server and agent to install. Options in defaults/manager_versions.yaml"),
-
-        dict(name="target_manager_version", env="SCT_TARGET_MANAGER_VERSION",
-             type=str,
-             help="Branch of scylla manager server and agent to upgrade to. Options in defaults/manager_versions.yaml"),
-
-        dict(name="manager_scylla_backend_version", env="SCT_MANAGER_SCYLLA_BACKEND_VERSION",
-             type=str,
-             help="Branch of scylla db enterprise to install. Options in defaults/manager_versions.yaml"),
-
-        dict(name="scylla_mgmt_agent_version", env="SCT_SCYLLA_MGMT_AGENT_VERSION", type=str,
-             help=""),
-
-        dict(name="scylla_mgmt_pkg", env="SCT_SCYLLA_MGMT_PKG",
-             type=str,
-             help="Url to the scylla manager packages to install for management tests"),
-
-        dict(name="stress_cmd_lwt_i", env="SCT_STRESS_CMD_LWT_I",
-             type=str,
-             help="Stress command for LWT performance test for INSERT baseline"),
-
-        dict(name="stress_cmd_lwt_d", env="SCT_STRESS_CMD_LWT_D",
-             type=str,
-             help="Stress command for LWT performance test for DELETE baseline"),
-
-        dict(name="stress_cmd_lwt_u", env="SCT_STRESS_CMD_LWT_U",
-             type=str,
-             help="Stress command for LWT performance test for UPDATE baseline"),
-
-        dict(name="stress_cmd_lwt_ine", env="SCT_STRESS_CMD_LWT_INE",
-             type=str,
-             help="Stress command for LWT performance test for INSERT with IF NOT EXISTS"),
-
-        dict(name="stress_cmd_lwt_uc", env="SCT_STRESS_CMD_LWT_UC",
-             type=str,
-             help="Stress command for LWT performance test for UPDATE with IF <condition>"),
-
-        dict(name="stress_cmd_lwt_ue", env="SCT_STRESS_CMD_LWT_UE",
-             type=str,
-             help="Stress command for LWT performance test for UPDATE with IF EXISTS"),
+Boolean = Annotated[bool, BeforeValidator(_boolean)]
 
-        dict(name="stress_cmd_lwt_de", env="SCT_STRESS_CMD_LWT_DE",
-             type=str,
-             help="Stress command for LWT performance test for DELETE with IF EXISTS"),
 
-        dict(name="stress_cmd_lwt_dc", env="SCT_STRESS_CMD_LWT_DC",
-             type=str,
-             help="Stress command for LWT performance test for DELETE with IF condition>"),
+def sct_field(*args, **kwargs):
+    kwargs.setdefault('default', None)
+    assert 'env' in kwargs
+    return Field(*args, **kwargs)
 
-        dict(name="stress_cmd_lwt_mixed", env="SCT_STRESS_CMD_LWT_MIXED",
-             type=str,
-             help="Stress command for LWT performance test for mixed lwt load"),
 
-        dict(name="stress_cmd_lwt_mixed_baseline", env="SCT_STRESS_CMD_LWT_MIXED_BASELINE",
-             type=str,
-             help="Stress command for LWT performance test for mixed lwt load baseline"),
+available_backends: list[str] = [
```

Comment:
this move away from the class, keeping any properties which are not part of pydantic model, create lots of boilerplate to find a way to ignore them during translation to yaml or other formats